crates/valence_protocol/src/lib.rs

```diff
@@ -337,52 +351,60 @@ pub trait DecodePacket<'a>: Sized + fmt::Debug {
 mod derive_tests {
     use super::*;
 
-    #[derive(Encode, EncodePacket, Decode, DecodePacket)]
+    #[derive(Debug, Encode, EncodePacket, Decode, DecodePacket)]
     #[packet_id = 1]
     struct RegularStruct {
         foo: i32,
         bar: bool,
         baz: f64,
     }
 
-    #[derive(Encode, EncodePacket, Decode, DecodePacket)]
+    #[derive(Debug, Encode, EncodePacket, Decode, DecodePacket)]
     #[packet_id = 2]
     struct UnitStruct;
 
-    #[derive(Encode, EncodePacket, Decode, DecodePacket)]
+    #[derive(Debug, Encode, EncodePacket, Decode, DecodePacket)]
     #[packet_id = 3]
     struct EmptyStruct {}
 
-    #[derive(Encode, EncodePacket, Decode, DecodePacket)]
+    #[derive(Debug, Encode, EncodePacket, Decode, DecodePacket)]
     #[packet_id = 4]
     struct TupleStruct(i32, bool, f64);
 
-    #[derive(Encode, EncodePacket, Decode, DecodePacket)]
+    #[derive(Debug, Encode, EncodePacket, Decode, DecodePacket)]
     #[packet_id = 5]
-    struct StructWithGenerics<'z, T = ()> {
+    struct StructWithGenerics<'z, T = ()>
+    where
+        T: std::fmt::Debug,
+    {
```

Comment:
unrelated changes

Reply:
I did but these changes should go in a separate PR to keep things focused.